### ISSUE TITLE
Allow usage of filename

### ISSUE THREAD

**Issue by schneiderfelipe** [insilichem/esigen#10]
Hi,

I tried to use `{{jobfilename}}` and `{{filename}}` within a template without success (reports an empty string). Any idea on how to do that?

cclib has them, but only on some situations I believe (an example with cclib where those attributes are actually used can be found [here](https://github.com/cclib/cclib/blob/808c1922da5b98db4863c0c9d8a42e6e2f164f6b/cclib/io/ccio.py#L350)).

And thanks for the great tool!

**Comment by jaimergp** [comment]
Hi! Thanks for showing interest in `esigen`!

Latest `cclib` version (1.6) has more fields in the [`metadata` attribute](https://cclib.github.io/data_notes.html#metadata), which includes `input_file_name`.

To access data inside `metadata`, use key access like this:

```
{{ metadata['input_file_name'] }}
```

You will have to update `cclib` in your local installation. I will update the demo webserver during the day to reflect these changes.

Cheers,
Jaime.

**Comment by jaimergp** [comment]
Update:

I have performed some tests with `cclib` 1.6 and apparently they don't populate those fields yet (not for Gaussian, at least). In the meantime, I have added two more fields:

   -  `{{ filename }}`: Filename, with extension.
   -  `{{ filepath }}`: Full path to the file.

You will be able to use them if you update to the development version:

```
pip install -U https://github.com/insilichem/esigen/archive/master.zip
```
